netbox/overlays/ocp-staging/externalsecrets/netbox-s3-creds.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "kubernetes-client.io/v1"
+kind: ExternalSecret
+metadata:
+  name: netbox-s3-creds
+spec:
+  backendType: secretsManager
+  data:
+    - key: cluster/ocp-prod/netbox/pgbackrest-s3-conf
```

Comment:
are you sure you want to use the same bucket for backups as ocp-prod?

Reply:
Actually, I should just remove this secret; you'll note in the `postgrescluster` that I'm not using s3 backups for staging (so this would have been a no-op).